recipes/xorg/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+
+
+class ConanXOrg(ConanFile):
+    name = "xorg"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    homepage = "https://www.x.org/wiki/"
+    description = "The X.Org project provides an open source implementation of the X Window System."
+    settings = {"os": "Linux"}
+
+    def package_id(self):
```

Comment:
I'm curious why header_only, if we can use only one configuration (os=Linux)? Is it necessary?

Reply:
to erase the `os` setting